os/drivers/lcd/mipi_lcd.c

```diff
@@ -349,5 +416,17 @@ FAR struct lcd_dev_s *mipi_lcdinitialize(FAR struct mipi_dsi_device *dsi, struct
 	}
 	priv->config->backlight(CONFIG_LCD_MAXPOWER);
 
+#if defined(CONFIG_LCD_SW_ROTATION)
+	for (int itr = 0; itr < NUM_OF_LCD_BUFFER; itr++) {
+		lcd_buffer[itr] = (uint8_t *)kmm_malloc(CONFIG_LCD_XRES * CONFIG_LCD_YRES * 2 + 1);
```

Comment:
How about allocating entire size at once instead of allocating each node?
for example, if 1 frame size is 100,
we can allocate 200 at once, and internally we can assign the address on each array.

Why I suggest this is
1. each allocation needs a chunk at start of the node. Its size is 16 byte. We can reduce it.
2. Because this allocation is done at booting time, there is no adv in possibility of allocation. 
    Because this allocation size is big, if we do this at middle of app operation, we could get failure on huge size allocation. But this will be ok with 1 huge node at booting time.